.storybook/preview.js

```diff
@@ -11,10 +11,30 @@ export const parameters = {
   },
 };
 
-export const decorators = [
-  (Story) => (
-    <ThemeProvider theme={defaultTheme}>
-      <Story />
+export const globalTypes = {
+  theme: {
+    name: 'Theme',
+    description: 'Global theme for components',
+    defaultValue: 'light',
+    toolbar: {
+      items: ['light', 'dark', 'blue'],
+      showName: true,
+    },
+  },
+};
+
+const withThemeProvider=(Story,context)=> {
```

Comment:
What do you think of something along these lines for readability?

```typescript

const withThemeProvider = (Story, context) => {
  const theme = getThemeFromContext(context);

  return (
    <ThemeProvider theme={theme}>
      <Story {...context} />
    </ThemeProvider>
  );
};

function getThemeFromContext(context) {
  switch (context?.globals?.theme) {
    case "dark":
      return dark;
    case "blue":
      return blue;
    default:
      return light;
  }
}
```

Reply:
Fixed by commit `e3f99457e7bb5a5179dcb23260e93a436e32eb85`

Reply:
If you copy the link of the commit and then paste, it will look like this https://github.com/Major-Eth/yearn-web-lib/pull/19/commits/e3f99457e7bb5a5179dcb23260e93a436e32eb85